ReactApp/src/main.jsx

```diff
@@ -1,5 +1,13 @@
+/**
+ * External dependencies
+ */
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+
+/**
+ * Internal dependencies
+ */
+import './misc/api-fetch-setup';
```

Comment:
I am generally (possibly unnecessarily) wary of side-effect modules. It can make automated testing difficult and limits how the utility can be applied. WDYT about implementing an initializer function instead? 

```js
import { initializeApiFetch } from './misc/api-fetch-setup';

initializeApiFetch();
```

Reply:
Good idea! I'll update it

Reply:
Added in https://github.com/wordpress-mobile/GutenbergKit/pull/10/commits/27fcf10bc372d2a7752c9df104e7283e1f3bc55d